_posts/events/2002-01-21-gwc.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+categories: [event]
+event_name: "The 1st Global Wordnet Conference 2002"
+event_url: "https://aclanthology.org/venues/gwc/"
+event_location: "Mysore, India"
+siglex_endorsed: yes
+---
+
+Proceedings at https://aclanthology.org/venues/gwc/
```

Comment:
Write this as "Proceedings at <https://aclanthology.org/venues/gwc/>" to make sure that a link is generated. Same comment for the 3 below.

Otherwise, this pull request looks great!

Reply:
Thank you, I will fix.